hack/local-up-cluster.sh

```diff
@@ -768,21 +767,69 @@ function start_kubelet {
           cred_bind="--volume=${cloud_cred}:${cloud_cred}:ro"
       fi
 
-      docker run \
-        --volume=/:/rootfs:ro \
+      docker run --rm --name kubelet \
+        --volume=/:/rootfs:ro,rslave \
         --volume=/var/run:/var/run:rw \
         --volume=/sys:/sys:ro \
-        --volume=/var/lib/docker/:/var/lib/docker:ro \
-        --volume=/var/lib/kubelet/:/var/lib/kubelet:rw \
+        --volume=/var/lib/docker/:/var/lib/docker:rslave \
+        --volume=/var/lib/kubelet/:/var/lib/kubelet:rslave \
         --volume=/dev:/dev \
         --volume=/run/xtables.lock:/run/xtables.lock:rw \
         ${cred_bind} \
         --net=host \
+        --pid=host \
         --privileged=true \
         -i \
         --cidfile=$KUBELET_CIDFILE \
         k8s.gcr.io/kubelet \
-        /kubelet --v=${LOG_LEVEL} --containerized ${priv_arg}--chaos-chance="${CHAOS_CHANCE}" --pod-manifest-path="${POD_MANIFEST_PATH}" --hostname-override="${HOSTNAME_OVERRIDE}" ${cloud_config_arg} \ --address="127.0.0.1" --kubeconfig "$CERT_DIR"/kubelet.kubeconfig --port="$KUBELET_PORT"  --enable-controller-attach-detach="${ENABLE_CONTROLLER_ATTACH_DETACH}" &> $KUBELET_LOG &
+        /kubelet ${priv_arg} \
+        --containerized \
+        --v=${LOG_LEVEL} \
```

Comment:
Would it be possible to populate some global variable with these common options and use it both in non-containerized and containerized kubelet? As we've seen, people add new options only to the non-containerized path and I am afraid the list of options will get out of sync quickly.